src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/subcommand/AboutSubcommand.java

```diff
@@ -38,12 +45,41 @@ public Integer call() throws IOException, SQLException {
         "This tool allows to extract meta information from a data warehouse that allows");
     System.out.println("to make an assessment for migration.");
     System.out.println();
-    try (InputStreamReader reader =
-        new InputStreamReader(this.getClass().getResource("/NOTICE.txt").openStream())) {
-      for (String s : CharStreams.readLines(reader)) {
-        System.out.println(s);
+    System.out.println("Data Warehouse Assessment Extraction Client");
+    System.out.println("Copyright 2021 Google LLC");
+    System.out.println();
+    List<String> dependencyNames =
+        Resources.readLines(
+            AboutSubcommand.class.getResource("/third_party/dependency-list.txt"), UTF_8);
+    for (String dependencyName : dependencyNames) {
+      Dependency dependency =
+          objectMapper.readValue(
+              Resources.toString(
+                  AboutSubcommand.class.getResource(
+                      "/third_party/" + dependencyName + "/meta.json"),
```

Comment:
This will fail if dependency is listed in the dependency-list.txt, but no folder is provided. What about the opposite scenario – do we want to check for unexpected folders?

Reply:
I've replaced the static dependency-list.txt file with a genrule that generates it during the build. This should make both scenarios much less likely.